### ISSUE TITLE
Crash: Unrecoverable error. basic_string::_S_create in io_realm_internal_table.cpp line 1433

### ISSUE THREAD

**Issue by diegomontoya** [realm/realm-java#1604]
Getting a few crash reports of the following. 

io.realm.exceptions.RealmError: Unrecoverable error. basic_string::_S_create in io_realm_internal_table.cpp line 1433
       at io.realm.internal.Table.nativeGetName(Table.java)
       at io.realm.internal.Table.getName(Table.java:1470)
       at io.realm.internal.Table.getPrimaryKey(Table.java:627)
       at io.realm.internal.Table.hasPrimaryKey(Table.java:653)
       at io.realm.Realm.checkHasPrimaryKey(Realm.java:1009)
       at io.realm.Realm.copyToRealmOrUpdate(Realm.java:760)

0) Using snapshot realm-android-0.83.1-20151012.100616-17
1) No encryption
2) Using only Armv7a librealm-jni.so
3) Realm Table/Object def

```java
    @PrimaryKey
    private long a;

    @Index
    private int b;

    @Index
    private int c;

    @Required
    private String d;

    @Index
    private int e;
```

4) Due to the very low crash rate of this bug, not possible to reproduce on our devices.

**Comment by kneth** [comment]
It is indeed a strange error. It seems that Realm cannot retrieve the name of the underlying table. What is the name of your model class? In under which locale do you see the error?

**Comment by diegomontoya** [comment]
The realmObject class is named "RealmReview" and the realmModule class is named "RealmModuleReview". Only ASCII chars are used in the names.

**Comment by diegomontoya** [comment]
May be related. 

This is the same realm instance/table that is experiencing the issue

https://github.com/realm/realm-java/issues/1610

**Comment by kneth** [comment]
Let us assume that they are related - and the fix for #1610 is also a fix for this one.

**Comment by kneth** [comment]
As #1610 has been closed, I close the issue. Don't hesitate to create a new or reopen of the issue persist.
